docs/scu/dev/how-the-spryker-code-upgrader-works.md

```diff
@@ -42,6 +42,13 @@ https://api.release.spryker.com/release-group/XXXX
 
 To continue running the Upgrader tool, install the major version manually, and re-run the Upgrader tool.
 
+{% info_block infoBox "Composer dependency conflict" %}
+
+Spryker code upgrader uses the [composer](https://getcomposer.org/) for updating modules and libraries. 
+If the composer detects a conflict, it stops the update process and returns the error message. To resolve the conflict, you need to manually update the conflicting module and re-run the Upgrader pipline.
```

Comment:
pipline -> pipeline

Reply:
If the composer detects a conflict, it stops the update process and generates an error message. To resolve the conflict, you must manually update the conflicting module and then re-run the Upgrader pipeline.

Reply:
Thanks ++